frontend/src/lib/components/LemonInput/LemonInput.scss

```diff
@@ -14,13 +20,22 @@
         text-overflow: ellipsis;
 
         &:disabled {
-            opacity: 0.6;
+            opacity: var(--opacity);
             cursor: not-allowed;
         }
     }
+
+    .LemonRow__icon:first-child {
+        font-size: 18px;
+        color: var(--text-muted);
+
+        svg.LemonRow__icon__icon--active {
+            color: var(--primary);
+        }
+    }
```

Comment:
`LemonInput` is based on `LemonRow` so it's okay to override styles for some `.LemonRow` classes here, but inventing new classes for the base component elsewhere is quite messy – otherwise you have to look at various SCSS files to find all the classes and `LemonRow` classes start to leak out to other TSX files.
I propose we add a modifier to `.LemonInput` instead and just do a child selector based on that, e.g.
```SCSS
.LemonRow__icon:first-child {
    .LemonInput & {
        font-size: 18px;
        color: var(--text-muted);
    }
    .LemonInput.LemonInput--with-input & {
        color: var(--primary);
    }
}
```